source/index-management.rst

```diff
@@ -689,25 +769,33 @@ Common API Components
     wildcardProjection: Document;
   }
 
-
-----------------------
-Backwards Compatibilty
-----------------------
-
-This specification makes no attempts to be backwards compatible as the target drivers to implement this spec are all next generation.
-
-
 ---------
 Q & A
 ---------
 
 Q: Where is write concern?
-  The createIndexes and dropIndexes commands take a write concern that indicates how the write is acknowledged. Since all operations defined in this specification are performed on a collection, it's uncommon that two different index operations on the same collection would use a different write concern. As such, the most natural place to indicate write concern is on the client, the database, or the collection itself and not the operations within it.
+  The ``createIndexes`` and ``dropIndexes`` commands take a write concern that indicates how the write is acknowledged. Since all operations defined in this specification are performed on a collection, it's uncommon that two different index operations on the same collection would use a different write concern. As such, the most natural place to indicate write concern is on the client, the database, or the collection itself and not the operations within it.
+
+  However, it might be that a driver needs to expose write concern to a user per operation for various reasons. It is permitted to allow a write concern option, but since writeConcern is a top-level command option, it MUST NOT be specified as part of an ``IndexModel`` passed into the helper. It SHOULD be specified via the options parameter of the helper, if there is one. For example, it would be ambiguous to specify write concern for one or more models passed to ``createIndexes()``, but it would not be to specify it via the ``CreateIndexesOptions``. If a helper does not have options defined for it in this specification, an appropriate options type SHOULD be defined for it, and write concern SHOULD be specified through those options. This allows for compatibility with any future options that may be required to be accepted by those helpers.
+
```

Comment:
This option was a little confusing to me at first, so I added a few questions that I wanted the answers to when I was thinking about how to document this option.